usmqe/api/tendrlapi/common.py

```diff
@@ -27,15 +25,23 @@ def get_job_attribute(self, job_id, attribute="status", section=None):
             section:    section of response in which is attribute located
         """
         pattern = "jobs/{}".format(job_id)
-        response = requests.get(pytest.config.getini("usm_api_url") + pattern)
+        response = requests.get(
+            pytest.config.getini("usm_api_url") + pattern,
+            headers={"Authorization": "Bearer {}".format(credentials["access_token"])})
```

Comment:
Well, but what if we want to test the case when the header is not present, and that's the reason why I propose that.